src/test/java/org/folio/calendar/integration/api/health/BaseApiInitializationTest.java

```diff
@@ -0,0 +1,16 @@
+package org.folio.calendar.integration.api.health;
```

Comment:
How come this class is in the health API package?

Reply:
No longer applicable due to the removal of this test.